pstress/pstress-run-PXC80.conf

```diff
@@ -133,16 +133,16 @@ SAVE_SQL=0
 STORE_COPY_OF_INFILE=0
 
```

Comment:
Remove "mohit.joshi" from WORKDIR. Change it to:
WORKDIR=$HOME/pxc_runs/$RANDOMD